client/checker_client/checker.py

```diff
@@ -419,12 +419,17 @@ def deploy_checker(
             print("  deployed: chunk {}.".format(chunk_no))
 
     print("Sealing.")
+
+    external_contracts = {
+        "oracle": oracle,
```

Comment:
I'm actually surprised that we can pass `ContractInterface` types here instead of the string with the address, but I guess pytezos must convert it internally. I'm happy leaving it since that is what we were doing before anyways.

Reply:
Right?! I could add the `.context.address` parts if you think important; otherwise feel free to merge :slightly_smiling_face:

Reply:
Ya no worries, it's what we were doing before anyways.